configure.py

```diff
@@ -548,13 +555,19 @@ def add_generated_header(h: str):
         n.build("generated_headers_" + version, "phony", generated_headers)
         n.newline()
 
-    for c_file in glob("src/**/*.c", recursive=True):
-        if c_file.endswith(".inc.c"):
-            continue
 
-        status = await shell_status(f"grep -q SCRIPT\( {c_file}")
+    for version in versions:
+        if version == "us":
+            c_files = glob("src/**/*.c", recursive=True)
+        elif version == "jp":
+            c_files = glob("ver/jp/src/**/*.c", recursive=True)
```

Comment:
are we just gonna not have jp load anything from src/ for now

Reply:
(imo it should read from both dirs, and should be the case for all versions)

Reply:
yeah I don't think it's worth it cause splat isn't set up to work with it anyway